tests/cluster/etl/test_loader.py

```diff
@@ -9,41 +9,100 @@
 
 class LoaderTests(unittest.TestCase):
     @classmethod
-    def setUpClass(cls):
+    def setUp(self):
```

Comment:
why not use setUpClass?

Reply:
It is because i validate if the functions have been called:
Save test:
self.assertTrue(self.loader.save.called)
self.assertFalse(self.loader.save_many.called)
Save_many test:
self.assertFalse(self.loader.save.called)
self.assertTrue(self.loader.save_many.called)

This would not work for setupClass, as the function would always be called

Reply:
Alright. At first look I imagined that the classes (and datasets) what setup too many times. But I see now.